develop/cli/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN git -c http.sslVerify=false clone --depth 1 -b $OPENRCT2_REF https://github.
  && mkdir build \
  && cd build \
  && cmake .. -G Ninja -DCMAKE_BUILD_TYPE=release -DCMAKE_INSTALL_PREFIX=/openrct2-install/usr -DDISABLE_GUI=ON -DENABLE_HEADERS_CHECK=OFF \
+ && ninja -k0 graphics \
```

Comment:
Curious, why was this fine before? That is, it wasn't explicitly invoking `ninja g2`, right?

Reply:
Good question. My guess is that the build process used to have some kind of dependency on the g2 target, but no longer does. I looked for clues in [the commit that broke these builds](https://github.com/OpenRCT2/OpenRCT2/commit/7ddb80606367aba8aa637213c0604712e4240683#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20a) but I'm not super familiar with how cmake works.

Reply:
Also this underlying issue doesn't only affect docker. I tried building the latest commit via Arch's openrct2 PKGBUILD and it also failed.

Reply:
As i said previously all we need to do is fix the cmake install to not install g2.dat/font.dat if not building gui

Reply:
@duncanspumpkin Non-GUI builds can still render screenshots, so they need g2.dat.